templates/ipfs/etc/systemd/system/ipfs.service

```diff
@@ -3,7 +3,6 @@ Description=IPFS daemon
 After=network.target
 
 [Service]
-Type=notify
```

Comment:
? ipfs does send a notification. This would make the service become ready before IPFS actually is.

Reply:
Fixed in: https://github.com/hsanjuan/ansible-ipfs-cluster/pull/10/commits/019c47f03edc6737de140ec754bc38af9055011a

I had a lot of trouble setting up the molecule tests with docker because of systemd. This was probably something that got left behind.